addons/website/static/src/snippets/s_facebook_page/000.js

```diff
@@ -24,37 +30,62 @@ const FacebookPageWidget = publicWidget.Widget.extend({
             params.href = `https://www.facebook.com/${params.id}`;
         }
         delete params.id;
-        params.width = utils.confine(Math.floor(this.$el.width()), 180, 500);
 
-        var src = $.param.querystring('https://www.facebook.com/plugins/page.php', params);
-        this.$iframe = $('<iframe/>', {
-            src: src,
-            width: params.width,
-            height: params.height,
-            css: {
-                border: 'none',
-                overflow: 'hidden',
-            },
-            scrolling: 'no',
-            frameborder: '0',
-            allowTransparency: 'true',
-        });
-        this.$el.append(this.$iframe);
+        this._renderIframe(params);
+
+        this.resizeObserver = new ResizeObserver(debounce(this._renderIframe.bind(this, params), 100));
+        this.resizeObserver.observe(document.querySelector("#wrapwrap"));
 
-        this.options.wysiwyg && this.options.wysiwyg.odooEditor.observerActive();
         return def;
     },
     /**
      * @override
      */
-    destroy: function () {
-        this._super.apply(this, arguments);
+    destroy() {
+        this._super(...arguments);
```

Comment:
[I asked you to change it in ES6](https://github.com/odoo/odoo/pull/163739#discussion_r1639440000) as it was a diff on those lines anyway, but now that there are no more diff I prefer you don't touch it...